SofascoreAcademy/SofascoreAcademy/View/LeagueEventsView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LeagueEventsView.swift
+//  SofascoreAcademy
+//
+//  Created by Anton Pomper on 17.03.2024..
+//
+
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class LeagueEventsView: BaseView {
+    
+    private let leagueView = LeagueView()
+    private let eventsContainerView = UIView()
+    
+    override func addViews() {
+        addSubview(leagueView)
+        addSubview(eventsContainerView)
+    }
+    
+    override func setupConstraints() {
+        leagueView.snp.makeConstraints { make in
+            make.top.left.right.equalToSuperview()
+        }
+        
+        eventsContainerView.snp.makeConstraints { make in
+            make.top.equalTo(leagueView.snp.bottom)
+            make.left.right.equalToSuperview()
+        }
+    }
+    
+    func configure(with leagueViewModel: LeagueViewModel, events: [EventModel]) {
+        leagueView.configure(with: leagueViewModel)
+        
+        var previousEventView: UIView? = nil
```

Comment:
cijelu ovu logiku stackanja jedno ispod drugog možete si olakšati tako da koristite stack view i u njega stavljate viewove, on će to automatski odraditi za vas